docs/cog_guides/audio.rst

```diff
@@ -3261,21 +3261,28 @@ manage an unmanaged (external) or managed Lavalink node.
     server to do so. Changing llset command settings have the potential to break 
     Audio cog connection and playback if the wrong settings are used.
 
-""""""""""""""
-llset external
-""""""""""""""
+"""""""""""""""
+llset unmanaged
+"""""""""""""""
 
 **Syntax**
 
 .. code-block:: none
 
-    [p]llset external
+    [p]llset unmanaged
+
+or 
+
+.. code-block:: none
+
+    [p]llsetup unmanaged
 
 **Description**
 
-Toggle using external Lavalink nodes - requires an existing external Lavalink node for
-Audio to work, if enabled. This command disables the managed Lavalink server: if you do
-not have an external Lavalink node you will be unable to use Audio while this is enabled.
+Toggle using unmanaged (user-managed) Lavalink nodes - requires an existing Lavalink 
+node for Audio to work, if enabled. This command disables the managed (bot-managed) 
+Lavalink server: if you do not have an unmanaged Lavalink node available, you will be 
```

Comment:
Instead of "...if you do not have an unmanaged Lavalink node **available**...", perhaps something like "set up", that better implies that it requires explicit user action (unlike the managed server)?